common/reviews/api/webpack-deep-imports-compat-plugin.api.md

```diff
@@ -4,28 +4,24 @@
 
 ```ts
 
-import type { Compiler } from 'webpack';
-import type { Configuration } from 'webpack';
-import type { WebpackPluginInstance } from 'webpack';
+import { Compiler } from 'webpack';
+import { DllPlugin } from 'webpack';
 
 // @public
-export class DeepImportsCompatPlugin implements WebpackPluginInstance {
+export class DeepImportsCompatPlugin extends DllPlugin {
+    constructor(options: IDeepImportsCompatPluginOptions);
     // (undocumented)
     apply(compiler: Compiler): void;
-    // (undocumented)
-    static applyToWebpackConfiguration(webpackConfiguration: Configuration, options: IDeepImportsCompatPluginOptions): void;
 }
 
+// Warning: (ae-forgotten-export) The symbol "DllPluginOptions" needs to be exported by the entry point index.d.ts
```

Comment:
Shouldn't this be exported?

Reply:
This type technically comes from Webpack. It feels wrong for us to re-export something that's really just an alias for a type from Webpack that isn't correctly exported.

Reply:
@TheLarkInn is Webpack every going to fix these typings? 😝